pkg/reconciler/inmemorychannel/dispatcher/testdata/config-event-dispatcher.yaml

```
@@ -0,0 +1,27 @@
+# Copyright 2020 The Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: config-imc-event-dispatcher
+  namespace: knative-eventing
+  labels:
+    eventing.knative.dev/release: devel
+data:
+  # The ConfigMapFromTestFile helper method expects a key named "_example" and it's placeholder here.
+  _example: |
+    placeholder: "nothing"
+  MaxIdleConnections: 20
+  MaxIdleConnectionsPerHost: 10
```

Let's stick to the usual `camelCase` format for ConfigMaps keys.
```suggestion
  maxIdleConnections: 20
  maxIdleConnectionsPerHost: 10
```

Also, if your example reflects the default values, we shouldn't need to redefine them here (cf. most core ConfigMaps such as logging, observability, etc.)

---

These values are not used anymore, let me remove these